### PR TITLE
Meta: Update `typed-query-selector`

### PR DESCRIPTION
**Pull request by fregante**
Follows:

- https://github.com/g-plane/typed-query-selector/pull/28
- https://github.com/g-plane/typed-query-selector/issues/26
- https://github.com/g-plane/typed-query-selector/issues/29

### CODE REVIEW

File: source/features/show-whitespace.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ const viewportObserver = new IntersectionObserver(changes => {
 	}
 });
 
-function showWhitespaceWhenInViewport(line: HTMLElement): void {
+function showWhitespaceWhenInViewport(line: Element): void {
```
Comment (fregante) [inline source/features/show-whitespace.tsx:20]: @g-plane I think your recent changes somehow broke this. It appeared as soon as I installed the update

Comment (g-plane) [reply source/features/show-whitespace.tsx:20]: It shouldn't be, because you're using your own type helper.

Comment (fregante) [reply source/features/show-whitespace.tsx:20]: See https://www.typescriptlang.org/play?target=2#code/JYWwDg9gTgLgBAchgTzAUwCYFoCOBXNKZLAZzQBs0BjGaBAbgChG0APSWOKiAOxPm4Y0AUUog0PGCQDKFarShwAvHADajAJAIAdACNyEXVkFoswHj0IIANHAD0duAGEIQuOXNoSmhAC4eEDAAFNoBMFAAhnzkETBoAJRwAHxwoYGR0bFoNvaOLm76EFQA1iRw5lwQIOKSJNpwALIRyHC6aHAA7pFg6BhwMB3AVGiMALpwEWXcfDBMzNP8cKzKcBhFeDUw2viEyLKUNNBBAAa+wCRBACQA3iaiaJsycodQ2gBWEOZBNgjxAL7xfyBEKFIwmLAACzwPGK8WO8SYcCRDjgAD0APxAA

It's `never` for some reason, likely causes by `.join()` returning a simple `string`

Comment (fregante) [reply source/features/show-whitespace.tsx:20]: You can change the playground version with a comment but it requires a reload to take effect. Here it is working with v2.8

Screenshot


and not working with v2.9.0

Screenshot 1

File: source/features/sort-conversations-by-update-time.tsx [hunk 2]
```diff
@@ -57,7 +56,7 @@ function updateLink(link: HTMLAnchorElement): void {
 function init(signal: AbortSignal): void {
 	// Get issues links that don't already have a specific sorting applied
 	observe(
-		css`
+		`
```
Comment (fregante) [inline source/features/sort-conversations-by-update-time.tsx:59]: I can restore this when I figure out how to preserve the exact string type:

- https://github.com/fregante/code-tag/issues/12

File: package.json [hunk 3]
```diff
@@ -112,7 +112,7 @@
 		"terser-webpack-plugin": "^5.3.6",
 		"ts-node": "^10.9.1",
 		"type-fest": "^3.5.7",
-		"typed-query-selector": "^2.8.0",
+		"typed-query-selector": "github:g-plane/typed-query-selector#6338b01",
```
Comment (fregante) [inline package.json:115]: CI only fails because this hasn't been published to npm yet

Comment (g-plane) [reply package.json:115]: You can upgrade to v2.9.1 now.

**Pull request merged**